watchman/views.py

```diff
@@ -35,6 +35,17 @@ def _deprecation_warnings():
         warnings.warn("`WATCHMAN_TOKEN` setting is deprecated, use `WATCHMAN_TOKENS` instead. It will be removed in django-watchman 1.0", DeprecationWarning)
 
 
+def _optional_apm_ignore_transaction():
+    if settings.WATCHMAN_DISABLE_APM:
+
+        # New Relic
+        try:
+            import newrelic.agent
+            newrelic.agent.ignore_transaction(flag=True)
```

Comment:
According to the newrelic docs, `flag=True` is the default - thoughts on dropping from here?